### ISSUE TITLE
Allow All rule for GKE VPC

### ISSUE THREAD

**Issue by backstreetkiwi** [stackabletech/t2#436]
@fhennig asked if the GCP GKE VPC could please have an "allow all rule" in order to access something from the outside.

Here are the corresponding Terraform docs:

https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/compute_network

Here is the line in our templates where the network is defined:

https://github.com/stackabletech/t2/blob/d4b3537d3a97229852a1e127e1050039c9cd6b14/templates/gke/main.tf#L46

**Comment by backstreetkiwi** [comment]
The rule was added to the GKE clusters and it worked when we configure a service of type LoadBalancer. The ticket will remain open until the devs tell me that it works.

**Comment by backstreetkiwi** [comment]
T2 has to be hotfixed with a 3.9.1 because of this change whic is currently only in the container:
```
# Allow all for external access in tests
resource "google_compute_firewall" "rules" {
  name                    = "${local.cluster_name}"
  network                 = "${local.cluster_name}"
  priority                = 0
  source_ranges           = ["0.0.0.0/0"]
  depends_on              = [ google_compute_network.vpc ]

  allow {
    protocol  = "all"
  }
}
```

**Comment by backstreetkiwi** [comment]
fixed in T2 3.10.0
